src/util/browser.js

```diff
@@ -1,97 +1,86 @@
-
-/**
- * @module browser
- * @private
- */
+// @flow
 
 const window = require('./window');
 
 /**
- * Provides a function that outputs milliseconds: either performance.now()
- * or a fallback to Date.now()
+ * @private
  */
-module.exports.now = (function() {
-    if (window.performance &&
-        window.performance.now) {
-        return window.performance.now.bind(window.performance);
-    } else {
-        return Date.now.bind(Date);
-    }
-}());
-
-const frame = window.requestAnimationFrame ||
-    window.mozRequestAnimationFrame ||
-    window.webkitRequestAnimationFrame ||
-    window.msRequestAnimationFrame;
-
-exports.frame = function(fn) {
-    return frame(fn);
-};
-
-const cancel = window.cancelAnimationFrame ||
-    window.mozCancelAnimationFrame ||
-    window.webkitCancelAnimationFrame ||
-    window.msCancelAnimationFrame;
+module.exports = {
+    /**
+     * Provides a function that outputs milliseconds: either performance.now()
+     * or a fallback to Date.now()
+     */
+    now: (() => {
+        if (window.performance &&
+            window.performance.now) {
+            return window.performance.now.bind(window.performance);
+        } else {
+            return Date.now.bind(Date);
+        }
```

Comment:
Why use the immediately invoked function expression (IIFE) here (and below in `frame`, `cancelFrame`) rather than declaring `const now = (window.performance && window.performance.now) ? window.performance.now.bind(window.performance) : Date.now.bind(Date);` prior to defining the exported object?

No strong objection to this, but all else being equal, I think IIFE is maybe a bit less readable.